galleries/examples/scales/custom_scale.py

```diff
@@ -6,12 +6,21 @@
 ============
 
 Create a custom scale, by implementing the scaling use for latitude data in a
```

Comment:
I'd like to start with scoping, e.g.:

> Custom scales can be used in two ways:
> 
> - use FuncScale (link?j for simple cases.
> - Create a custom scale class when you need more control, such as
>
>    [your bullet list here]
> 
> This example shows the latter, by implementing the scaling use for latitude data in a Mercator Projection.

Reply:
I re-ordered your suggestion a little, because I wanted to introduce what we are doing (Mercator Projection) before mentioning the Mercator transform in the first sub-bullet.